doc/tutorials/enable_s5.rst

```diff
@@ -204,72 +268,69 @@ How to Test
-in the Service VM, which is connected to ``ttyS1`` in the pre-launched VM. The
-lifecycle manager in the pre-launched VM receives the shutdown command, sends an
+In the ``hybrid_rt`` scenario, operator can use the script to send a system shutdown
+request via ``/var/lib/life_mngr/monitor.sock`` to User VM which is configured to be allowed to
+trigger system S5, this system shutdown request is forwarded to the Service VM, the
+Service VM sends poweroff request to each User VMs (Pre-launched VM or Post-launched VM)
+through vUART. The Lifecycle Manager in the User VM receives the poweroff request, sends an
 ack message, and proceeds to shut itself down accordingly.
 
 .. figure:: images/system_shutdown.png
    :align: center
 
    Graceful system shutdown flow
 
-#. The HMI Windows Guest uses the lifecycle manager to send a shutdown request to
-   the Service VM
-#. The lifecycle manager in the Service VM responds with an ack message and
-   executes ``s5_trigger.sh``
-#. After receiving the ack message, the lifecycle manager in the HMI Windows Guest
-   shuts down the guest
-#. The ``s5_trigger.sh`` script in the Service VM shuts down the Linux Guest by
-   using ``acrnctl`` to send a shutdown request
-#. After receiving the shutdown request, the lifecycle manager in the Linux Guest
-   responds with an ack message and shuts down the guest
-#. The ``s5_trigger.sh`` script in the Service VM shuts down the Pre-launched RTVM
-   by sending a shutdown request to its ``ttyS1``
-#. After receiving the shutdown request, the lifecycle manager in the Pre-launched
-   RTVM responds with an ack message
-#. The lifecycle manager in the Pre-launched RTVM shuts down the guest using
-   standard PM registers
-#. After receiving the ack message, the ``s5_trigger.sh`` script in the Service VM
-   shuts down the Service VM
-#. The hypervisor shuts down the system after all of its guests have shut down
+#. The HMI in the Windows VM uses ``s5_trigger_win.py`` to send
+   system shutdown request to the Lifecycle Manager, Lifecycle Manager
+   forwards this request to Lifecycle Manager in the Service VM.
+#. The Lifecycle Manager in the Service VM responds with an ack message and
+   sends ``poweroff_cmd`` request to Windows VM.
+#. After receiving the ``poweroff_cmd`` request, the Lifecycle Manager in the HMI
+   Windows VM responds with an ack message, then shuts down VM.
+#. The Lifecycle Manager in the Service VM sends ``poweroff_cmd`` request to
+   Linux User VM.
+#. After receiving the ``poweroff_cmd`` request, the Lifecycle Manager in the
+   Linux User VM responds with an ack message, then shuts down VM.
+#. The Lifecycle Manager in the Service VM sends ``poweroff_cmd`` request to
+   Pre-launched RTVM.
+#. After receiving the ``poweroff_cmd`` request, the Lifecycle Manager in
+   the Pre-launched RTVM responds with an ack message.
+#. The Lifecycle Manager in the Pre-launched RTVM shuts down the VM using
+   standard PM registers.
```

Comment:
Can we rephrase this somehow? What does it mean to "use the standard PM registers", is this a traditional shutdown as if you were running on bare-metal hardware?

Reply:
Thanks, it is ACPI PM register, will update this.